extension/snippets/dvc-yaml.code-snippets

```diff
@@ -0,0 +1,55 @@
+{
+  "Get Started with DVC pipelines": {
+    "prefix": "dvc-help",
+    "body": [
```

Comment:
Minor! Some of these sentences are written "in the passive voice" which is recommended to avoid in writing. Here's a list of comparison sentences, but feel free to ignore them if they sound worse.

```
Snippets are provided by the DVC for VS Code extension to help with building dvc.yaml pipelines.

The DVC for the VSCode extension provides snippets to help with building dvc.yaml pipelines.
```

```
For further details on building pipelines visit https://dvc.org/doc/user-guide/project-structure/dvcyaml-files

Visit https://dvc.org/doc/user-guide/project-structure/dvcyaml-files for further details on building pipelines.
```

```
If the redhat.yaml extension is installed then on-hover support will be provided for each of the keys in a dvc.yaml file.

On-hover support will be provided for each of the keys in a dvc.yaml file if the redhat.yaml extension is installed.
```